Sources/ApolloCodegenLib/Templates/SwiftPackageManagerModuleTemplate.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+struct SwiftPackageManagerModuleTemplate {
+  let moduleName: String
+
+  func render() -> String {
+    TemplateString("""
+    // swift-tools-version:5.3
+
+    import PackageDescription
+
+    let package = Package(
+      name: "\(moduleName)",
+      platforms: [
+        .iOS(.v12),
+        .macOS(.v10_14),
+        .tvOS(.v12),
+        .watchOS(.v5),
+      ],
+      products: [
+        .library(name: "\(moduleName)", targets: ["\(moduleName)"]),
+      ],
+      targets: [
+        .target(name: "\(moduleName)"),
```

Comment:
Questionable whether we should have a `test` target? We _could_ provide some basic sanity tests for the generated code but compilation _should_ be enough.

Reply:
On second thought maybe sanity tests of the generated queries to ensure we're providing all expected type casts isn't a bad idea?

Reply:
We shouldn't be adding generated tests to our user's projects. We need to be testing that everything works properly and generates all valid code in the Apollo tests, but generated code should just be assumed to work.

Reply:
We do need to add dependencies on ApolloAPI here though!

Reply:
> We do need to add dependencies on ApolloAPI here though!

Dang, I completely forgot about that!

Reply:
@AnthonyMDev, the manifest now pulls in `ApolloAPI` - https://github.com/apollographql/apollo-ios/pull/2120/commits/4e2cb16a1e646e28bc766f710a7f5b5094b3886a.